lib/quickbooks/service/preferences.rb

```diff
@@ -3,8 +3,7 @@ module Service
     class Preferences < BaseService
 
       def url_for_query(query = nil, start_position = 1, max_results = 20, options = {})
-        url = super(query, start_position, max_results, options)
-        "#{url}&minorversion=#{Quickbooks::Model::Preferences::MINORVERSION}"
+        super(query, start_position, max_results, options)
       end
```

Comment:
Does this method need to be here? Seems like it could be removed and just let super hand it.

Reply:
@drewish you're totally right, these are unnecessary. Thanks!